arm-iothub/2017-01-19/swagger/iothub.json

```diff
@@ -0,0 +1,1962 @@
+      "description": "Identity registry statistics.",
+      "type": "object",
+      "properties": {
+        "totalDeviceCount": {
+          "format": "int64",
+          "description": "The total count of devices in the identity registry.",
+          "type": "integer",
+          "readOnly": true
+        },
+        "enabledDeviceCount": {
+          "format": "int64",
+          "description": "The count of enabled devices in the identity registry.",
+          "type": "integer",
+          "readOnly": true
+        },
+        "disabledDeviceCount": {
+          "format": "int64",
+          "description": "The count of disabled devices in the identity registry.",
+          "type": "integer",
+          "readOnly": true
+        }
+      }
+    },
+    "JobResponseListResult": {
+      "description": "The JSON-serialized array of JobResponse objects with a next link.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The array of JobResponse objects.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/JobResponse"
+          }
+        },
+        "nextLink": {
+          "description": "The next link.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "IotHubSkuDescription": {
+      "description": "SKU properties.",
+      "type": "object",
+      "properties": {
+        "resourceType": {
+          "description": "The type of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "sku": {
+          "$ref": "#/definitions/IotHubSkuInfo"
+        },
+        "capacity": {
+          "$ref": "#/definitions/IotHubCapacity"
+        }
+      },
+      "required": [
+        "sku",
+        "capacity"
+      ]
+    },
+    "IotHubCapacity": {
+      "description": "IoT Hub capacity information.",
+      "type": "object",
+      "properties": {
+        "minimum": {
+          "format": "int64",
+          "description": "The minimum number of units.",
+          "type": "integer",
+          "minimum": 1,
+          "maximum": 1,
+          "readOnly": true
+        },
+        "maximum": {
+          "format": "int64",
+          "description": "The maximum number of units.",
+          "type": "integer",
+          "readOnly": true
+        },
+        "default": {
+          "format": "int64",
+          "description": "The default number of units.",
+          "type": "integer",
+          "readOnly": true
+        },
+        "scaleType": {
+          "description": "The type of the scaling enabled.",
+          "enum": [ "Automatic", "Manual", "None" ],
+          "readOnly": true,
+          "type": "string",
+          "x-ms-enum": {
+            "name": "IotHubScaleType",
+            "modelAsString": false
+          }
+        }
+      }
+    },
+    "EventHubConsumerGroupsListResult": {
+      "description": "The JSON-serialized array of Event Hub-compatib
```

Comment:
Seems like the comments went to the wrong place. Sorry about that. 

I was referring to line number 1928 `"security": [ ]`.